### PR TITLE
Add website icon

### PR DESCRIPTION
**Pull request by mukeshmandiwal**

### CODE REVIEW

**Comment by mukeshmandiwal** [comment]
fixes #375

**Comment by crashuniverse** [comment]
We can remove the 2nd commit that adds delectation message by rebasing with latest branch. The PR is easier to focus if it has only your contribution.

File: docs/src/Root.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ const Root = props => {
     <>
       <Helmet>
         <title>{title}</title>
+        <link rel="shortcut icon" type="image/png" href="/logo.png"/>
```
Comment (crashuniverse) [inline docs/src/Root.js]: Logo in this case is 512 size, maybe a smaller 64x64 file for favicon.png or favigon.ico would be a better usage than logo.

**Comment by karlhorky** [comment]
Maybe we should have a background color for dark mode? So that the favicon doesn't show black on dark gray like GitHub?

Screen Shot 2019-04-09 at 11 17 07

**Comment by mukeshmandiwal** [comment]
Should we use logo-white.png as favicon ?

**Comment by karlhorky** [comment]
Hm, that will have problems on non-dark mode tabs, wouldn't it?

I guess my suggestion would be to fill in the transparent parts as well as a 2 pixel margin around it with white (2 pixels when it's shrunk to favicon size).

Something that would look like this:

**Comment by karlhorky** [comment]
You may also want to consider generating some other versions of the favicon with something like RealFaviconGenerator (you could take a look at my PR here: keepassxreboot/keepassxreboot.github.io#54)

File: docs/src/index.js [hunk 2]
```diff
@@ -134,6 +134,7 @@ const Page = props => {
     <>
       <Helmet>
         <title>{title}</title>
+        <link rel='shortcut icon' type='image/x-icon' href='/favicon.ico'/>
```
Comment (jxnblk) [inline docs/src/index.js]: Thanks for looking into this! Whatever you end up using to generate the icon with, can you please add an `npx` script to the `package.json`? – see how repng is used for the other images

Comment (mukeshmandiwal) [reply docs/src/index.js]: I use [https://realfavicongenerator.net/](url) link to generate favicon from static/logo.svg file

Comment (jxnblk) [reply docs/src/index.js]: Thanks! If there’s not an installable package this will likely fall out of sync over time

Comment (mukeshmandiwal) [reply docs/src/index.js]: As of my concern, there is no installable package which likely fulfills our requirements

Comment (karlhorky) [reply docs/src/index.js]: Maybe one of the following npm packages? I just did a quick search for "svg to ico npm" on Google:

- https://www.npmjs.com/package/favicons
- https://www.npmjs.com/package/icon-gen

Comment (jxnblk) [reply docs/src/index.js]: It could also be generated from the `png` output from repng, if that makes finding a library easier

Comment (jxnblk) [reply docs/src/index.js]: Is there any reason *not* to use a `.png` file for the favicon? I assume most modern browsers support this now, but haven't done any research

**Comment by jxnblk** [comment]
Thanks again for the work on this! I'll go ahead and merge this PR since I'm getting ready to ship updated docs for v5

**Pull request merged**